src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,6 +32,62 @@
  */
 class OgSelection extends DefaultSelection {
 
+  /**
+   * The OG access service.
+   *
+   * @var \Drupal\og\OgAccessInterface
+   */
+  protected $OgAccess;
+
+  /**
+   * The OG membership manager.
+   *
+   * @var \Drupal\og\MembershipManagerInterface
+   */
+  protected $OgMembershipManager;
```

Comment:
Protected properties should start with a small cap.

```suggestion
  protected $ogMembershipManager;
```